src/applications/gi/containers/SearchPage.jsx

```diff
@@ -191,6 +191,7 @@ export class SearchPage extends React.Component {
                 cautionFlag={result.cautionFlag}
                 studentCount={result.studentCount}
                 bah={result.bah}
+                dodBah={result.dodBah}
```

Comment:
Is this prop necessary if we aren't actually using it in the SearchResult component?